cs/testing.texy

```diff
@@ -449,6 +449,19 @@ demo/
 
 Název adresáře můžeme změnit v `Tester\Dumper::$dumpDir`.
 
+Přeskakování testů
+==================
+Můžeme mít i testy, které nejdou spustit. Třeba se píšeme knihovnu, co používá jazykovou konstrukci, ale ta je až v nějakém PHP a ve starší verzi chceme ten test prostě přeskočit. Uděláme to takto:
```

Comment:
"co používá" -> "která používá"
"je až v nějakém PHP" "je dostupná až od určité verze PHP"